lib/init.coffee

```diff
@@ -10,4 +14,44 @@ module.exports =
         type: 'string'
 
   activate: ->
-    console.log 'activate linter-gjslint'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-gjslint.executablePath',
+      (executablePath) =>
+        @executablePath = executablePath
+  deactivate: ->
+    @subscriptions.dispose()
+  provideLinter: ->
+    helpers = require('atom-linter')
+    provider =
+      name: 'gjslint'
+      grammarScopes: ['source.js', 'source.js.jquery', 'text.html.basic']
+      scope: 'file'
+      lintOnFly: true
+      lint: (editor) =>
+        filePath = editor.getPath()
+        cwd = path.dirname(filePath)
+        tempFile path.basename(filePath), editor.getText(), (tmpFilePath) =>
+          params = [
+            '--nobeep',
+            '--quiet',
+            tmpFilePath
+          ]
+          return helpers.exec(@executablePath, params, {cwd}).then (stdout) ->
+            regex = /^Line\s(\d*)\, E\:(\d*):\s(.*)$/
+            lines = stdout.split('\n').filter (line) ->
+              line.indexOf('Line') == 0
+
+            return [] unless lines.length
+            return lines.map (msg) ->
+              res = regex.exec(msg)
+              [all, line, code, text] = res
```

Comment:
It would be nice to include the code in your message, to make looking up the error simpler if the user is unfamiliar with it or it is not showing results they are expecting.